source/db/en-projects.json

```diff
@@ -3655,7 +3655,7 @@
     "description": "A safe home for all your data",
     "license_url": "https://github.com/nextcloud/server/blob/master/COPYING-README",
     "logo": "nextcloud.png",
-    "notes": "Nextcloud provides a file sync & share solution that respects your privacy and is fully open source. Sync clients are available for all major platforms. Nextcloud comes with a rich ecosystem of apps including Contacts, Calendar, Video calls, Mail, Newsreader and much more.",
+    "notes": "Nextcloud provides a file sync & share solution that respects your privacy and is fully open source. Sync clients are available for all major platforms. Nextcloud comes with a rich ecosystem of apps including Contacts, Calendar, Video calls, Mail, Newsreader and much more.\n\nNextcloud originated as a fork of ownCloud. It is preferred because ownCloud has an \"open core\" product model and lacks some security features Nextcloud has.",
```

Comment:
I'm on mobile so I can't check, but isn't this interpolated directly into the HTML? So I _think_ these `\n`s should be `<br />`s or `</p><p>` or something.